client/state/ui/guided-tours/test/contexts.js

```diff
@@ -8,9 +8,10 @@ import { expect } from 'chai';
  */
 import useMockery from 'test/helpers/use-mockery';
 import useFakeDom from 'test/helpers/use-fake-dom';
+import { EDITOR_PASTE_EVENT_FROM_GDOCS } from 'state/action-types';
```

Comment:
I learned today that Calypso's `action-types` now weighs 7 KB gzip'd! That makes me want to make this action type more reusable and not open a precedent for similar uses creating new action types (e.g. other pastes, maybe triggers for certain embeds, etc.). So:

```js
// off the top of my mind:
{ type: EDITOR_PASTE_EVENT, pasteType: 'google-docs' }
// or more drastic:
{ type: PASTE_EVENT, source: 'editor', pasteType: 'google-docs' }
```